katello/katello-restore

```diff
@@ -36,9 +36,9 @@ def restore
   puts "Done.\n"
 
   puts "Restoring backend data"
-  `tar --selinux --overwrite -xf pulp_data.tar -C /`
-  `tar --selinux --overwrite -xzf mongo_data.tar.gz -C /`
-  `tar --selinux --overwrite -xzf pgsql_data.tar.gz -C /`
+  `tar --selinux --overwrite --listed-incremental=pulp.snar -xf pulp_data.tar -C /`
```

Comment:
@sean797 - Users should not be restoring from a backup where pulp content was skipped (outlined in the new documentation still awaiting merge). --skip-pulp option would only be utilized for things like debugging. I'll go ahead and fail the restore if the backup file is missing.